migration-engine/migration-engine-tests/compile-bench.md

```diff
@@ -1,3 +1,3 @@
 | Command | Mean [s] | Min [s] | Max [s] | Relative |
 |:---|---:|---:|---:|---:|
-| `cargo build --tests` | 5.590 ± 0.073 | 5.491 | 5.713 | 1.00 |
+| `cargo build --tests` | 6.508 ± 0.049 | 6.420 | 6.584 | 1.00 |
```

Comment:
I was a bit surprised. Less code, and it should be lighter to compile. Maybe we added enough tests since the last bench run that it compensates the effect, or maybe there's a been a compiler performance regression.